zomes/holo_acorn/code/src/lib.rs

```diff
@@ -127,9 +127,13 @@ pub struct ArchiveGoalResponse {
     archived_goal_votes: Vec<Address>,
     archived_goal_comments:Vec<Address>,
 }
-
-// The GetResponse struct allows our zome functions to return an entry along with its
-// address so that Redux can know the address of goals and edges
+#[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
+pub struct GetHistoryResponse {
+    entrys: Option<Vec<Goal>>,
```

Comment:
what about?
```suggestion
    entries: Vec<Goal>,
```

why Option?

Reply:
it could just be a 0 length Vec